client/components/vertical-menu/style.scss

```diff
@@ -22,8 +22,12 @@
 
 	&.is-selected {
 		border-left: 3px solid $blue-wordpress;
-		color: darken( $gray, 30% );
+		color: $gray-dark;
```

Comment:
Yea, I think we can merge my 5 or so PRs first, then switch the variable names in #11368